src/utils.ts

```diff
@@ -51,7 +52,7 @@ export interface CacheProvider {
 
 export function getCacheProvider(): CacheProvider {
   const cacheProvider = core.getInput("cache-provider");
-  const cache = cacheProvider === "github" ? ghCache : cacheProvider === "buildjet" ? buildjetCache : undefined;
+  const cache = cacheProvider === "github" ? ghCache : cacheProvider === "buildjet" ? buildjetCache : cacheProvider === "blacksmith" ? bsCache : undefined;
```

Comment:
should we just add it as the _only_ provider?

Reply:
should we keep them while we benchmark ours against theirs?

Reply:
good idea